library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/DdiProviderDoctrineRepository.php

```diff
@@ -73,11 +73,11 @@ public function findByBrandAndProxyTrunks(BrandInterface $brand, ProxyTrunkInter
 
     /**
      * @param ProxyTrunkInterface $proxyTrunks
-     * @return array
+     * @return DdiProviderInterface[]
      */
-    public function findByProxyTrunks(ProxyTrunkInterface $proxyTrunks)
+    public function findByProxyTrunks(ProxyTrunkInterface $proxyTrunks): ?array
     {
-        /** @var DdiProviderInterface[] $response */
+        /** @var DdiProviderInterface[]|null $response */
```

Comment:
Doctrine says this will never return null
```
    /** {@inheritDoc} */
    public function findBy(array $criteria, ?array $orderBy = null, ?int $limit = null, ?int $offset = null): array
```

Change `: ?array` above as well